docker/generic/start_proxy.py

```diff
@@ -189,6 +189,11 @@ def make_argparser():
        backward compatible for ESPv1 and will be deprecated.
        Please use the flag --listener_port.''')
 
+    parser.add_argument('-S', '--ssl_port', default=None, type=int, help='''
+    Expose a port for HTTPS requests.  Accepts both HTTP/1.x and HTTP/2
+    secure connections. Requires the certificate and key files
+    /etc/endpoints/ssl/envoy.crt and /etc/endpoints/ssl/envoy.key''')
```

Comment:
not need to mention "envoy",  how about server.crt, and server.key

We also need to add one or two flags to allow user to specify different path/name

Reply:
We can definitely add two flags for cert and key. But since ESPv1 doesn't do this, we can always require API producers to put the cert/key into the right path. 

We can follow up this later.